src/Emulator/Peripherals/Peripherals/Mocks/HPSHostController.cs

```diff
@@ -325,6 +321,43 @@ private byte[] GetBytesFromSlave(int count, TimeInterval timeInterval)
             return result;
         }
 
+        // This method is for getting OA1EN, 
+        // returns true when OA1EN is enabled.
+        private bool OA1ENEnabled(){
+            return currentSlave.ReadDoubleWord(0x08) >> 15 != 0x00;
```

Comment:
In 4d678705d3b90ba65125bea27520e31c37758bd2 we exposed additional properties in the I2C model, so now it's possible to just write:
```
   return currentSlave.OwnAddress1Enabled;
```

Reply:
Done